doc/nrf/known_issues.rst

```diff
@@ -1006,13 +1006,22 @@ NCSDK-6832: SMP Server sample fails upon initialization
 SEGGER Embedded Studio Nordic Edition
 *************************************
 
-.. rst-class:: v1-5-0 v1-4-2 v1-4-1 v1-4-0
+.. rst-class:: v1-4-2 v1-4-1 v1-4-0
```

Comment:
Is this change on purpose? Since it's not mentioned in the commit.

Reply:
it's a followup from this PR:
https://github.com/nrfconnect/sdk-nrf/pull/3771#discussion_r580095034

@umapraseeda prefered to just get #3771 in, and then I removed the `v1-5-0` in this PR I had to do anyway.